tests/integration-tests.bats

```diff
@@ -33,5 +33,11 @@ load ../script/libtest
 @test "Test function: post-nodejs" {
   test_kubeless_function post-nodejs
 }
-
+@test "Test function: pubsub" {
+  skip "XXX(jjo): flaky kafka->zookeeper under minikube, likely from its kube-dns"
```

Comment:
See e.g. for **same** build https://travis-ci.org/jjo/kubeless/builds/264162697
* OK -> https://travis-ci.org/jjo/kubeless/jobs/264162698
* Failed -> https://travis-ci.org/jjo/kubeless/jobs/264162699, at log
```
pods/kafka-0: org.I0Itec.zkclient.exception.ZkException: Unable to connect to zookeeper.kubeless:2181
pods/kafka-0: 	at org.I0Itec.zkclient.ZkConnection.connect(ZkConnection.java:72)
pods/kafka-0: 	at org.I0Itec.zkclient.ZkClient.connect(ZkClient.java:1228)
pods/kafka-0: 	at org.I0Itec.zkclient.ZkClient.<init>(ZkClient.java:157)
pods/kafka-0: 	at org.I0Itec.zkclient.ZkClient.<init>(ZkClient.java:131)
pods/kafka-0: 	at kafka.utils.ZkUtils$.createZkClientAndConnection(ZkUtils.scala:106)
pods/kafka-0: 	at kafka.utils.ZkUtils$.apply(ZkUtils.scala:88)
pods/kafka-0: 	at kafka.server.KafkaServer.initZk(KafkaServer.scala:326)
pods/kafka-0: 	at kafka.server.KafkaServer.startup(KafkaServer.scala:187)
pods/kafka-0: 	at kafka.server.KafkaServerStartable.startup(KafkaServerStartable.scala:39)
pods/kafka-0: 	at kafka.Kafka$.main(Kafka.scala:67)
pods/kafka-0: 	at kafka.Kafka.main(Kafka.scala)
pods/kafka-0: Caused by: java.net.UnknownHostException: zookeeper.kubeless: Name or service not known
```
It's that dns resolution failure ^^^ what I found to be flaky (required
by kafka -> zoo service build up), thus skipping that test until we
can debug why minikube flakes there :/